Update getting start on OpenXR instructions (#511)
MicrosoftDocs/mixed-reality: mixed-reality-docs/mr-dev-docs/develop/native/openxr-getting-started.md

To start developing OpenXR applications for immersive Windows Mixed Reality headsets:

> 1. Be sure you're running at least the Windows 10 October 2020 Update (20H2), which is the minimum requirement for Windows Mixed Reality end users to run OpenXR applications.  If you're on an earlier version of Windows 10, you can upgrade by using the <a href="https://www.microsoft.com/software-download/windows10" target="_blank">Windows 10 Update Assistant</a>.

Reviewer: ```suggestion
1. Be sure you're running at least the Windows 10 October 2020 Update (20H2), which is the minimum supported OS release for Windows Mixed Reality end users to run OpenXR applications.  If you're on an earlier version of Windows 10, you can upgrade by using the <a href="https://www.microsoft.com/software-download/windows10" target="_blank">Windows 10 Update Assistant</a>.
```

> 1. Be sure you're running at least the Windows 10 October 2020 Update (20H2), which is the minimum supported OS release for Windows Mixed Reality end users to run OpenXR applications.  If you're on an earlier version of Windows 10, you can upgrade by using the <a href="https://www.microsoft.com/software-download/windows10" target="_blank">Windows 10 Update Assistant</a>.

Reviewer: ```suggestion
1. Be sure you're running at least the Windows 10 October 2020 Update (20H2), which is the minimum supported OS release for Windows Mixed Reality end users to run OpenXR applications.  Earlier versions of Windows 10 should still work with OpenXR but may not have ideal performance or quality.  If you're on an earlier version of Windows 10, you can upgrade by using the <a href="https://www.microsoft.com/software-download/windows10" target="_blank">Windows 10 Update Assistant</a>.
```

2. [Set up a Windows Mixed Reality headset](../../../enthusiast-guide/set-up-windows-mixed-reality) or [enable the Windows Mixed Reality simulator](../advanced-concepts/using-the-windows-mixed-reality-simulator.md).

That's it!  The Windows Mixed Reality OpenXR runtime is installed and made active automatically for all Windows Mixed Reality users.  The Microsoft Store then keeps the runtime up to date.

> If you've been using multiple VR headsets on your PC, sometimes the active OpenXR runtime maybe changed.  To reactivate the Windows Mixed Reality OpenXR Runtime, launch "Mixed Reality Portal" from the Start menu, and select "Fix it" at the top of the window.  If that button is missing, the OpenXR runtime is already active.<br>

Reviewer 2: ```suggestion
If you've been using multiple VR headsets on your PC, keep in mind that the active OpenXR runtime may periodically change.  To reactivate the Windows Mixed Reality OpenXR Runtime, on the **Start** menu, launch "Mixed Reality Portal" and then select "Fix it" at the top of the window.  If that button is missing, the OpenXR runtime is already active.<br>
```

## Getting the OpenXR Developer Tools for Windows Mixed Reality

> When you are developing OpenXR applications for HoloLens 2 or PC VR, you might find this <a href="https://www.microsoft.com/store/productId/9n5cvvl23qbt" target="_blank">OpenXR Developer Tools for Windows Mixed Reality app</a> app handy for you.  This app provides a demo of various OpenXR features, along with a System Status page with key information about the active runtime and current headset.

Reviewer 3: We were saying that we will rename the app to "OpenXR Tools for Windows Mixed Reality'" in the release. Also perhaps remove the "When you are developing OpenXR applications" since we are trying to widen the audience.

> This app provides a demo of various OpenXR features, along with a System Status page with key information about the active runtime and current headset

Also need to add "This app also lets you customize some settings when running OpenXR applications, such as the render scale (on PC only), and lets you enabled the performance overlay."

Author: If we have a doc for the tool itself, it can say it's more than just developing apps.  In this page, the context is different.  We are talking about "if you are developing openxr apps, this tool can help".  I think this statement will remain true.

We don't have to make the app name change fore the name change is released to the store.

> When you are developing OpenXR applications for HoloLens 2 or PC VR, you might find this <a href="https://www.microsoft.com/store/productId/9n5cvvl23qbt" target="_blank">OpenXR Developer Tools for Windows Mixed Reality</a> app handy for you.  This app provides a demo of various OpenXR features, along with a System Status page with key information about the active runtime and current headset.

Reviewer 2: ```suggestion
If you're developing OpenXR applications for HoloLens 2 or PC VR, you might find this <a href="https://www.microsoft.com/store/productId/9n5cvvl23qbt" target="_blank">OpenXR Developer Tools for Windows Mixed Reality</a> app useful.   It provides a demo of various OpenXR features and a System Status page that contains key information about the active runtime and current headset.
```

> When you are developing OpenXR applications for HoloLens 2 or PC VR, you might find this <a href="https://www.microsoft.com/store/productId/9n5cvvl23qbt" target="_blank">OpenXR Developer Tools for Windows Mixed Reality app</a> app handy for you.  This app provides a demo of various OpenXR features, along with a System Status page with key information about the active runtime and current headset.

Author: ```suggestion
When you are developing OpenXR applications for HoloLens 2 or PC VR, you might find this <a href="https://www.microsoft.com/store/productId/9n5cvvl23qbt" target="_blank">OpenXR Developer Tools for Windows Mixed Reality</a> app handy for you.  This app provides a demo of various OpenXR features, along with a System Status page with key information about the active runtime and current headset.
```

> On HoloLens 2, you can simply go to the Store app and search for "OpenXR" to find this app to install.

Reviewer 2: ```suggestion
To find and install the Developer Tools app, on your HoloLens 2, go to the Store app and search for "OpenXR."
```

> When using the HoloLens 2 emulator, the easiest way to install the OpenXR Developer Tools for Windows Mixed Reality is through the [Windows Device Portal](../advanced-concepts/using-the-windows-device-portal.md). Navigating to the "OpenXR" page and then clicking the "Install" button under "Developer Features", which also works on physical HoloLens 2 devices.

Reviewer 2: ```suggestion
When using the HoloLens 2 emulator, the easiest way to install the OpenXR Developer Tools for Windows Mixed Reality is through the [Windows Device Portal](../advanced-concepts/using-the-windows-device-portal.md). In the Portal, navigate to the "OpenXR" page and then, under **Developer Features**, select the **Install** button. This also works on physical HoloLens 2 devices.
```

To start developing OpenXR applications for immersive Windows Mixed Reality headsets:

1. Be sure you're running at least the Windows 10 October 2020 Update (20H2), which is the minimum supported OS release for Windows Mixed Reality end users to run OpenXR applications.  Earlier versions of Windows 10 should still work with OpenXR but may not have ideal performance or quality.  If you're on an earlier version of Windows 10, you can upgrade by using the <a href="https://www.microsoft.com/software-download/windows10" target="_blank">Windows 10 Update Assistant</a>.
2. [Set up a Windows Mixed Reality headset](../../../enthusiast-guide/set-up-windows-mixed-reality) or [enable the Windows Mixed Reality simulator](../advanced-concepts/using-the-windows-mixed-reality-simulator.md).

That's it!  The Windows Mixed Reality OpenXR runtime is installed and made active automatically for all Windows Mixed Reality users.  The Microsoft Store then keeps the runtime up to date.

> If you've been using multiple VR headsets on your PC, keep in mind that the active OpenXR runtime may periodically change.  To reactivate the Windows Mixed Reality OpenXR Runtime, on the **Start** menu, launch "Mixed Reality Portal" and then select "Fix it" at the top of the window.  If that button is missing, the OpenXR runtime is already active.<br>

Author: ```suggestion
If you've been using multiple VR headsets on your PC, keep in mind that the active OpenXR runtime may be changed by another vendor's system software.  To reactivate the Windows Mixed Reality OpenXR Runtime, on the **Start** menu, launch "Mixed Reality Portal" and then select "Fix it" at the top of the window.  If that button is missing, the OpenXR runtime is already active.<br>
```
